src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -128,7 +128,7 @@ const attributeDelimiterHtml = layoutManager.tv ? '' : '<span class="hide">: </s
                 }
                 attributes.push(createAttribute(globalize.translate('MediaInfoInterlaced'), (stream.IsInterlaced ? 'Yes' : 'No')));
             }
-            if (stream.AverageFrameRate || stream.RealFrameRate) {
+            if ((stream.AverageFrameRate || stream.RealFrameRate) && stream.Type === 'Video') {
```

Comment:
This check seems necessary - the embedded cover image has `Level: -99` and `Framerate: 90000`.

Reply:
Removed.